### PR TITLE
Add xmvn debug logs

### PR DESCRIPTION
**Pull request by mwperina**
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: java-client-kubevirt.spec.in [hunk 1]
```diff
@@ -76,7 +76,7 @@ make %{?_smp_mflags}
 # necessary because jdk 1.8 comes as default with xmvn
 export JAVA_HOME="%{_java_jdk_home}"
 
-%mvn_build -i -- %{?_mvn_opts}
+%mvn_build -i -X -d -- %{?_mvn_opts}
```
Comment (arso) [inline java-client-kubevirt.spec.in:79]: What does -d switch does?  I cannot see that option in xmvn --help  (v3.0.0)

**Review by arso — COMMENTED** [review]
When I tried running xmvn with -d it failed because this option was not recognized. Perhaps it's a version specific parameter?

File: java-client-kubevirt.spec.in (hunk 1 above)
Comment (mwperina) [reply java-client-kubevirt.spec.in:79]: Full debugging: https://docs.fedoraproject.org/en-US/java-packaging-howto/manpage_mvn_build/

Comment (arso) [reply java-client-kubevirt.spec.in:79]: Ah right - thats from the macro itself rather than from xmvn :+1:

**Pull request merged**